tests/tests_test_workflow/test_perf_workflow/data/bundle_manifest.yml

```diff
@@ -0,0 +1,73 @@
+build:
```

Comment:
There is a `bundle_manifest.yaml` in [perf-test/data](https://github.com/opensearch-project/opensearch-build/blob/main/tests/tests_test_workflow/test_perf_workflow/perf_test/data/bundle_manifest.yaml), can we use either one of it instead of both?

Reply:
Without looking specifically at this one, generally I try reduce test data to a minimum and commit next to the test to avoid leftover data files if we move/delete the test, for example.

Reply:
Seems like an unintended change caused by **merge from main**. It wasn't part of original [commit](https://github.com/opensearch-project/opensearch-build/pull/639/commits/d9a888741b66f805815bac56423ff9b0f27a0db9). Good catch @VachaShah.
I have refactored the test_run files under respective test types